### PR TITLE
fixup! changed: remove internal sftp support

### PR DESCRIPTION
**Pull request by Rechi**

### CODE REVIEW

File: xbmc/utils/URIUtils.cpp [hunk 1]
```diff
@@ -968,8 +968,7 @@ bool URIUtils::IsInternetStream(const CURL& url, bool bStrictCheck /* = false */
 
   // Special case these
   if (url.IsProtocol("ftp") || url.IsProtocol("ftps")  ||
-      url.IsProtocol("dav") || url.IsProtocol("davs")  ||
-      url.IsProtocol("sftp"))
+      url.IsProtocol("dav") || url.IsProtocol("davs"))
```
Comment (notspiff) [inline xbmc/utils/URIUtils.cpp]: why would you remove this?

Comment (Rechi) [reply xbmc/utils/URIUtils.cpp]: If this needs to be checked the addon has to be asked, but don't assume anything in Kodi.

Comment (notspiff) [reply xbmc/utils/URIUtils.cpp]: well i won't merge this before you do the necessary code rewrites.

Comment (Rechi) [reply xbmc/utils/URIUtils.cpp]: @notspiff I could add a todo note and leave code like it is

Comment (notspiff) [reply xbmc/utils/URIUtils.cpp]: yes. it's not like this was done for rtmp for instance..

Comment (Rechi) [reply xbmc/utils/URIUtils.cpp]: updated

**Pull request merged**